spec/controllers/users_controller_spec.rb

```diff
@@ -30,6 +30,20 @@
     end
   end
 
+  describe "#me" do
+    let(:user) { valid_user }
+    context "when it is a valid request", valid_request: true do
+      it "gets single user information" do
+        get :me, id: user.id, format: :json
```

Comment:
I tried using  ` get 'users/me', to: 'users#me'` too it didn't work at the time I tried, I didn't know about rails resolution by precedence so I had it below the `resources users` this would actually be most preferred method